### PR TITLE
Css userq

### PR DESCRIPTION
**Pull request by naremanhilles**

### CODE REVIEW

**Review by hshahwan — COMMENTED** [review]
- solve your Travis
- don't push not necessary files in the same PR

File: public/css/user.css [hunk 1]
```diff
@@ -79,4 +79,71 @@
...
+    }
+    .userq-contanier{
+        padding: 10px;
+        float: left;
+        margin-top: 40px;
+        margin-left: 50px;
+        margin-bottom: 40px;
+        height: 430px;
+        width: 54%;
+        min-width: 300px;;
```
Comment (tomduggan85) [inline public/css/user.css:99]: @naremanhilles extra semicolon

File: public/css/cons.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    text-decoration: none;
+    color: #A3BBF8;
+    justify-content: center;
+    margin-left:22px;
+    margin-bottom: 2px;
+
+}
+.nav__main:hover, .nav__prev:hover, .nav__logout:hover{
+    border-bottom: 3px solid #A3BBF8;
+}
```
Comment (tomduggan85) [inline public/css/cons.css:32]: @naremanhilles You could reduce some of the duplicated styles with .nav__main, .nav__prev and .nav__logout using another selector similar to the one on line 30:

```
.nav__main, .nav__prev, .nav__logout {
  text-decoration: none;
  justify-content: center;
  margin-bottom: 2px;
}
```

Another alternative is to add a common "nav__item" class to all of those elements (in addition to the class they already have, like nav__main or nav__logout).  Then all of the styles could go into this common "nav__item" class, and another benefit is that you could add additional nav items in the future and they would look correct once you gave them the nav__item class, without you having to add anything to your css file.

File: public/css/user.css [hunk 3]
```diff
@@ -79,4 +79,71 @@
...
+        flex-direction: column;
+    }
+    .userq-title{
+        font-size: 40px;
+        color: #333333;
+        margin: 20px 50px 20px 50px;
+        font-family: 'Jomhuria', cursive;
+    }
+    .userq-form{
+        margin: 20px 50px 20px 50px;
```
Comment (tomduggan85) [inline public/css/user.css:113]: @naremanhilles You could shorten this to `margin: 20px 50px;` if you want to save a few characters 😄

File: src/controller/index.js [hunk 4]
```diff
@@ -26,5 +26,4 @@ router.post('/login', loginValidation, whoExist, home.loginPost);
 
 router.get('/userPrevQ', getToken, renderAllPosts, userPrevQ.get);
 router.get('/userMyProfile', getToken, renderMyPosts, userMyProfile.get);
-router.post('/signup', signupValidation, home.signupPost);
```
Comment (tomduggan85) [inline src/controller/index.js:29]: @naremanhilles Did you mean to remove this signup post handler?

File: src/controller/middleware/renderMyPosts.js [hunk 5]
```diff
@@ -1,10 +1,17 @@
...
+      .then((result) => {
+
+        req.result = result.rows;
+        next();
+      })
+      .catch((e) => {
+        next(e);
+      });
+  } else {
+    next();
```
Comment (tomduggan85) [inline src/controller/middleware/renderMyPosts.js:15]: @naremanhilles I wonder if this `else` should be an error (i.e. return a 401)

File: src/views/userQ.hbs [hunk 6]
```diff
@@ -1,13 +1,14 @@
-<section>
-    <div>
-        <span>طلب استشارة مجانية</span>
-        <form>
-            <label for="qTitle">عنوان الاستشارة</label>
-            <input type="text" name="qTitle" id="qTitle">
-            <label for="qContent">موضوع الاستشارة</label>
-            <input type="text" name="qContent" id="qContent">
+<section class="userq">
+    <h2>hi</h2>
```
Comment (tomduggan85) [inline src/views/userQ.hbs:2]: 👋

File: public/css/user.css (hunk 1 above)
Comment (naremanhilles) [reply public/css/user.css:99]: it's typo error ,already fixed :+1:

File: public/css/user.css (hunk 3 above)
Comment (naremanhilles) [reply public/css/user.css:113]: thank you tom,i edit it :+1:

File: src/controller/index.js (hunk 4 above)
Comment (naremanhilles) [reply src/controller/index.js:29]: yes becaues the same rout repeats at the same file :+1:

File: src/controller/middleware/renderMyPosts.js (hunk 5 above)
Comment (naremanhilles) [reply src/controller/middleware/renderMyPosts.js:15]: You are right :+1:   else {
    res.redirect('/');
  } now code correct.

File: src/views/userQ.hbs (hunk 6 above)
Comment (naremanhilles) [reply src/views/userQ.hbs:2]: this typo error ,thank you tom :+1:

File: src/controller/index.js (hunk 4 above)
Comment (tomduggan85) [reply src/controller/index.js:29]: Ah, got it!

File: src/controller/middleware/renderMyPosts.js (hunk 5 above)
Comment (tomduggan85) [reply src/controller/middleware/renderMyPosts.js:15]: Cool, thanks for doing that!
